tests/acceptance/bootstrap/SearchContext.php

```diff
@@ -36,33 +38,26 @@ class SearchContext implements Context {
 	private FeatureContext $featureContext;
 
 	/**
-	 * @When user :user searches for :pattern using the WebDAV API
-	 * @When user :user searches for :pattern and limits the results to :limit items using the WebDAV API
-	 * @When user :user searches for :pattern using the WebDAV API requesting these properties:
-	 * @When user :user searches for :pattern and limits the results to :limit items using the WebDAV API requesting these properties:
-	 * @When user :user searches for :pattern inside folder :scope using the WebDAV API
-	 * @When user :user searches for :pattern inside folder :scope in space :spaceName using the WebDAV API
-	 *
 	 * @param string $user
 	 * @param string $pattern
 	 * @param string|null $limit
+	 * @param string|null $scopeType
 	 * @param string|null $scope
 	 * @param string|null $spaceName
 	 * @param TableNode|null $properties
 	 *
-	 * @return void
+	 * @return ResponseInterface
+	 * @throws GuzzleException|JsonException
 	 */
-	public function userSearchesUsingWebDavAPI(
-		string    $user,
-		string    $pattern,
-		?string   $limit = null,
-		?string   $scope = null,
-		?string   $spaceName = null,
+	private function searchFiles(
+		string  $user,
+		string  $pattern,
+		?string $limit = null,
+		?string  $scopeType = null,
+		?string  $scope = null,
+		?string  $spaceName = null,
```

Comment:
```suggestion
	private function searchFiles(
		string $user,
		string $pattern,
		?string $limit = null,
		?string $scopeType = null,
		?string $scope = null,
		?string $spaceName = null,
```

Reply:
just aligning the indentations to match with  that of  `TableNode $properties = null` on line 59. I think same is followed in other methods too.